src/raftstore/store/config.rs

```diff
@@ -134,15 +144,15 @@ impl Default for Config {
             snap_gc_timeout: ReadableDuration::hours(4),
             messages_per_tick: 4096,
             max_peer_down_duration: ReadableDuration::minutes(5),
-            max_leader_missing_duration: ReadableDuration::hours(2),
+            max_leader_missing_duration: max_leader_missing_duration,
             snap_apply_batch_size: ReadableSize::mb(10),
             lock_cf_compact_interval: ReadableDuration::minutes(10),
             lock_cf_compact_bytes_threshold: ReadableSize::mb(256),
             // Disable consistency check by default as it will hurt performance.
             // We should turn on this only in our tests.
             consistency_check_interval: ReadableDuration::secs(0),
             report_region_flow_interval: ReadableDuration::minutes(1),
-            raft_store_max_leader_lease: ReadableDuration::secs(9),
+            raft_store_max_leader_lease: raft_store_max_leader_lease,
```

Comment:
it is dangerous to use the different timeout in different TiKVs.

Reply:
ping

Reply:
It does not change actually, I need tick and election timeout for calculating allowed_leader_missing_duration, it looks clear to put them together,